core/src/main/java/nl/nn/adapterframework/unmanaged/SpringJmsConnector.java

```diff
@@ -155,7 +141,7 @@ private void configureEndpointConnection() throws ConfigurationException {
 			log.debug("{} setting no transaction manager", this::getLogPrefix);
 		}
 		if (sessionTransacted) {
-			jmsContainer.setSessionTransacted(sessionTransacted);
+			jmsContainer.setSessionTransacted(true);
 		}
```

Comment:
Waarom doen we hier niet altijd onconditioneel `jmsContainer.setSessionTransacted(sessionTransacted);`?

Reply:
Goed punt. Ik fix het.